addons/sale_management/models/sale_order.py

```diff
@@ -152,7 +152,7 @@ def action_confirm(self):
         res = super(SaleOrder, self).action_confirm()
         for order in self:
             if order.sale_order_template_id and order.sale_order_template_id.mail_template_id:
-                self.sale_order_template_id.mail_template_id.send_mail(order.id)
+                self.sale_order_template_id.mail_template_id.with_user(SUPERUSER_ID).send_mail(order.id)
```

Comment:
Oh, by the way, please change the self to order, it seems to be a problem as well :sweat_smile:
```suggestion
                order.sale_order_template_id.mail_template_id.with_user(SUPERUSER_ID).send_mail(order.id)
```